vumi_http_proxy/test/test_clickme.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+# from click.testing import CliRunner
+from vumi_http_proxy import clickme
+from twisted.trial import unittest
+from twisted.trial.unittest import SynchronousTestCase
+
+# Testing segment
+
+
+class TestClickMe(unittest.TestCase):
+    def test_click(self):
+        # runner = CliRunner()
+        # import pdb; pdb.set_trace()
+        result = SynchronousTestCase.patch(
+                                           clickme.cli, clickme.cli.port, 8000)
```

Comment:
You want to patch the `main` method on `Initialize`. Something like:
```python
self.patch(http_proxy.Initialize, 'main', ... thing to replace main with ...)
```
Since TestClickMe inherits from `TestCase`, which eventually inherits from `SynchronousTestCase`, `patch` is availables as `self.patch`.